porch/controllers/rootsyncset/api/v1alpha1/rootsyncset_types.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// RootSyncSetSpec defines the desired state of RootSyncSet
+type RootSyncSetSpec struct {
+	ClusterRefs []*ClusterRef `json:"clusterRefs,omitempty"`
+	Template    *RootSyncInfo `json:"template,omitempty"`
```

Comment:
So we expect no variation between clusters?

Reply:
As I understand the plan, we're starting here and then going to figure out what variation we need.

Reply:
Yeah, no variation is a starting point. We know that for some (maybe many or all) cases we do need more than this.